examples/99-advanced/atomic-orbitals.py

```diff
@@ -0,0 +1,226 @@
+#    \end{equation}
+#
+# See `Hydrogen atom <https://en.wikipedia.org/wiki/Hydrogen_atom>`_ for more
+# details.
+#
+# This dataset evaluates this function for the hydrogen orbital 3dxy, with the
+# following quantum numbers:
+#
+# * Principal quantum number: ``n=3``
+# * Azimuthal quantum number: ``l=2``
+# * Magnetic quantum number: ``m=-2``
+
+grid = examples.load_hydrogen_orbital(3, 2, -2)
+grid
+
+
+###############################################################################
+# Plot the Orbital
+# ~~~~~~~~~~~~~~~~
+# Plot the orbital using :func:`add_volume() <pyvista.Plotter.add_volume>` and
+# using the default scalars contained in ``grid``, ``real_hwf``. This way we
+# can plot more than just the probability of the election, but also the phase
+# of the electron.
+#
+# .. note::
+#    Since the real value of evaluated wave function for this orbital varies
+#    between ``[-<value>, <value>]``, we cannot use the default opacity
+#    ``opacity='linear'``. Instead, we use ``[1, 0, 1]`` since we would like
+#    the opacity to be proportional to the absolute value of the scalars.
+
+pl = pv.Plotter()
+vol = pl.add_volume(grid, cmap='magma', opacity=[1, 0, 1])
+vol.prop.interpolation_type = 'linear'
+pl.camera.zoom(1.5)
+pl.show_axes()
+pl.show()
+
+
+###############################################################################
+# Plot the Orbital Contours as an Isosurface
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Generate the contour plot for the orbital by determining when the orbital
+# equals 10% the maximum value of the orbital. This effectively captures the
+# majority of the "volume" the electron potentially exists in.
+#
+# Note how we use the absolute value of the scalars when evaluating
+# :func:`contour() <pyvista.PolyDataFilters.contour>`. Otherwise, we would only
+# capture the location of the spin-up electron.
+
+eval_at = grid['real_hwf'].max() * 0.1
+contours = grid.contour(
+    [eval_at],
+    scalars=np.abs(grid['real_hwf']),
+    method='marching_cubes',
+)
+contours = contours.interpolate(grid)
+contours.plot(
+    categories=2,
+    smooth_shading=True,
+    annotations={-eval_at: 'down-spin', eval_at: 'up-spin'},
+    scalar_bar_args={'n_labels': 0, 'title': ''},
+)
+
+
+###############################################################################
+# Volumetric Plot: Plot the Orbitals using RGBA
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Let's now combine some of the best parts of the two above plots. The
+# volumetric plot is great for showing the probability of the "electron cloud"
+# orbitals, but the colormap doesn't quite match reality as well as the
+# isosurface plot.
+#
+# For this example we're going to use an RGBA colormap to tightly control way
+# the orbitals are plotted. For this, the opacity will be mapped to the
+# probability of the electron being at a location in the grid, which we can do
+# by taking the absolute value of the orbital's "hydrogen wave function". We
```

Comment:
Absolute value _squared_, but it looks like that's indeed what you're computing here (not suggesting an edit because wrapping would need shifting around).

I'd also omit "hydrogen" and the quotes, you really are looking at the (hydrogen) orbital's wave function. Of course you might mean that "hydrogen wave function" as in `real_hwf`, in which case I'd also recommend renaming `real_hwf`...